app/models/paper.rb

```diff
@@ -10,6 +10,12 @@ class Paper < ApplicationRecord
   validates_associated :authors
   validate :allowed_biases, :allowed_methodologies
 
+  before_save :downcase_name
+
+  def downcase_name
+    self.publication.downcase!
+  end
```

Comment:
We could probably also just do `before save ->(paper) {paper.publication.downcase!}` or the proc version `before_save proc {|paper| paper.publication.downcase!}`

Or we could just leave it, I don't really care either way